Command/PopulateCommand.php

```diff
@@ -90,11 +91,16 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $index = $input->getOption('index');
         $type = $input->getOption('type');
         $reset = !$input->getOption('no-reset');
+        $delete = !$input->getOption('no-delete');
+
         $options = array(
+            'delete'        => $delete,
```

Comment:
@thanosp `default` options are defined in `configureOptions`
see https://github.com/FriendsOfSymfony/FOSElasticaBundle/blob/master/Provider/AbstractProvider.php#L118